tests/compile.sh

```diff
@@ -122,12 +122,6 @@ else
   fi
 fi
 
-if [[ "${MAKE_OPT}" == *"32BIT=Y"* ]]; then
```

Comment:
Remove -DCCPP=ON from CMAKE_FLAGS

Reply:
Thanks, will do. For this particular change in `compile.sh`, I will only rerun the tests for one of the GNU platform to make sure the logic still works as expected (it will, because `CCPP` is no longer an option in any of the `CMakeLists.txt`).

Waiting to hear back about the wcoss_dell_p3 issue if more changes are needed before this.

Reply:
Done.